packages/api-gateway/src/drivers/http/routes/usuarios/schema/updateUsuarioSchema.js

```diff
@@ -0,0 +1,86 @@
+const {
+  usuario,
+  persona,
+  general,
+} = require('./generalSchema');
+
+const updateUsuarioSchema = {
+  tags: ['Usuario'],
+  description: 'Given an object with user required data and userId, then update user in database.',
+  title: 'updateUsuarioSchema',
+  params: {
+    type: 'object',
+    properties: {
+      usuarioId: usuario.usuarioId,
+    },
+    required: ['usuarioId'],
+  },
+  body: {
+    type: 'object',
+    properties: {
+      rolId: usuario.rolId,
+      usuario: usuario.usuario,
+      correo: usuario.correo,
+      estatus: usuario.estatus,
+      actualizado: usuario.actualizado,
+      persona: {
```

Comment:
You already have a file for persona schema use it here

Reply:
It's a little different the request schema than the response schema. How can I merge that schema?